.travis.yml

```diff
@@ -44,9 +46,20 @@ install:
   # Build OpenSDS Controller source code
   - make all
 
-script:
-  - ./install/CI/coverage
-  - ./install/CI/test
+

```

Comment:
how is this supposed to work ? series or parallel ? single failure stops all next steps ? in clouddragon etc., we have

1. ability to define some steps as mandatory, some as optional (to PASS, an optional FAILURE will NOT break the build)
2. ability to enable/disable steps to reduce build time (very useful when close to release)
3. ability to run sequential (predecessor/successor model) or parallel
please check.

Reply:
In Travis file different jobs are defined.
if we want specific jobs, we can run by passing job name
and all these jobs run parallel
![CIJobs](https://user-images.githubusercontent.com/48289385/62345847-7dbb2900-b511-11e9-8643-5cf7cac4acda.PNG)